CAF: sensor_manager: Add sensor manager sample for multiple cores (#7391)
nrfconnect/sdk-nrf: samples/caf_sensor_manager/README.rst

The workload simulator is a custom module used by this sample only.
This module handles the data received from the :ref:`caf_sensor_data_aggregator`, simulates workload on data using :c:func:`k_busy_wait` and notifies the aggregator module when the aggregator buffer can be freed.

> If you are running this sample on an SoC with multiple cores, the workload simulator module (``workload_sim``) is placed on the second core.

Author: Do not think so. We do not distinguish between cores.

> In case of running sample on SoC with multiple cores the workload simulator module (``workload_sim``) is placed on second core.
> All communication between the cores is done using :ref:`event_manager_proxy` and :ref:`IPC`.

Reviewer: ```suggestion
If you are running this sample on an SoC with multiple cores, the workload simulator module (``workload_sim``) is placed on the second core.
All communication between the cores is done using :ref:`event_manager_proxy` and :ref:`IPC`.
```
